packages/rosaenlg-lib/src/LanguageFrench.ts

```diff
@@ -341,9 +345,11 @@ export class LanguageFrench extends LanguageImpl {
       }
 
       if (sentenceParams.negative) {
-        modifiedVerbalGroup.negativeAdverb = sentenceParams.negativeAdverb || 'pas';
+        modifiedVerbalGroup.negativeAdverb = sentenceParams.negativeAdverb ?? 'pas';

```

Comment:
This allow to pass an empty string as negativeAdverb

Reply:
Yes. Please add an explicit comment in the code so that we don't change it in the future.

Reply:
done